pkg/webhooks/regularuser/regularuser.go

```diff
@@ -26,12 +28,15 @@ const (
 	customDomainGroup string = "managed.openshift.io"
 	netNamespaceKind  string = "NetNamespace"
 	netNamespaceGroup string = "network.openshift.io"
+	// Enforce for the following namespaces, regardless of what's configured in MCC
```

Comment:
```suggestion
	// Enforce for the following namespaces, regardless of what's configured in managed-cluster-config
```

Why isn't this `netNamespaceNS` also in `hookconfig`?

Reply:
My thinking was that namespaces listed in `hookconfig` are global and unknown the webhooks. The `netNamespaceNS` namespaces seem necessary in all cases (regardless of what `managed-cluster-config` says) to enforce proper net-namespace behavior based on [the existing unit tests](https://github.com/openshift/managed-cluster-validating-webhooks/blob/master/pkg/webhooks/regularuser/regularuser_test.go#L586-L620), so I hard-coded them into this specific hook to ensure that behavior is preserved.

Reply:
Removed the hard-coded namespaces from the `regularuser` webhook after discussing with @dustman9000, who implemented the net-namespaces enforcement logic for the hook.